lib/index.js

```diff
@@ -8,6 +8,7 @@ var Spawn = require("child_process").spawn
   , Ul = require("ul")
   , OneByOne = require("one-by-one")
   , Sliced = require("sliced")
+  , Q = require('q')
```

Comment:
@krzkaczor Sure! Probably on Sunday I will do it. :smile: :+1: Btw, is it possible to make a translator to translate COBOL into JS? See #5.